projects/Mallard/src/services/logging.ts

```diff
@@ -110,6 +114,8 @@ class Logging extends AsyncQueue {
                 userData.membershipData.contentAccess.digitalPack) ||
             false
         const iAP = iapReceipt ? true : false
+        const selectedEdition = await getSelectedEditionSlug()
+        const defaultEdition = await getDefaultEditionTitle()
```

Comment:
would it make sense to use the either the slug or the title for both selected/default?